### PR TITLE
Add jQuery from Django admin before the init js

### PR DESCRIPTION
**Pull request by 153957**
The init js requires jQuery, so ensure jQuery is always included before it is.

### CODE REVIEW

**Comment by 153957** [comment]
Todo: update readme to indicate admin is required installed app?

File: djrichtextfield/widgets.py [hunk 1]
```diff
@@ -30,7 +30,11 @@ def __init__(self, attrs=None, field_settings=None):
     @property
     def media(self):
         js = settings.CONFIG['js']
-        js.append(reverse(self.INIT_URL))
+        js.extend([
```
Comment (jaap3) [inline djrichtextfield/widgets.py]: +=

File: djrichtextfield/widgets.py [hunk 2]
```diff
@@ -30,7 +30,11 @@ def __init__(self, attrs=None, field_settings=None):
...
+            'admin/js/vendor/jquery/jquery.min.js',
+            'admin/js/jquery.init.js',
+            reverse(self.INIT_URL))
```
Comment (jaap3) [inline djrichtextfield/widgets.py]: extra )

File: README.rst [hunk 3]
```diff
@@ -21,7 +21,7 @@ Quickstart
 ----------
 
 Install ``django-richtextfield`` and add it to your Django
-project's ``INSTALLED_APPS``::
+project's ``INSTALLED_APPS``, ``django.contrib.admin`` must also be in ``INSTALLED_APPS``::
 
     INSTALLED_APPS += 'djrichtextfield'
```
Comment (jaap3) [inline README.rst]: INSTALLED_APPS = [
    'django.contrib.admin',
    ... 
    'djrichtextfield'
]

**Pull request merged**